experimental/dds/tree2/docs/undo/v1-undo.md

```diff
@@ -290,10 +294,12 @@ We can address this by using the following implementation strategy:
     for each piece of repair data associated with changesets that make up the transaction,
     leave the repair data in the `StygianForest` untouched but update its entry so that it is now associated with the composed changeset.
 
+Note that the `AnchorSet` for the view also needs to be patched so that any anchors pointing to the repair data is updated with the appropriate path.
```

Comment:
```suggestion
Note that the `AnchorSet` for the view also needs to be patched so that any anchors pointing to the repair data are updated with the appropriate path.
```